### PR TITLE
fixes(autok3s):Minor fixes for CI Process

### PR DESCRIPTION
**Pull request by orangedeng**
1. Replace sqlite driver to remove CGO support. CGO in build will be remove in the feature.
2. Update commit checker image as the latest image has been moved to new registry.
3. Using tags to inject embed ui files with conditions. When running locally, it won't require ui static files under ui package.

### CODE REVIEW

File: hack/make-rules/autok3s.sh [hunk 1]
```diff
@@ -275,20 +150,17 @@ function unit() {
 
   local os="${OS:-$(go env GOOS)}"
   local arch="${ARCH:-$(go env GOARCH)}"
+  local race_tag="-race"
   if [[ "${arch}" == "arm" ]]; then
     # NB(thxCode): race detector doesn't support `arm` arch, ref to:
     # - https://golang.org/doc/articles/race_detector.html#Supported_Systems
+    race_tag=""
+  fi
     GOOS=${os} GOARCH=${arch} CGO_ENABLED=1 go test \
```
Comment (JacieChao) [inline hack/make-rules/autok3s.sh:159]: Do we also need to DISABLE CGO here for the test?

File: Dockerfile.dapper [hunk 2]
```diff
@@ -1,62 +1,42 @@
...
+    mv /tmp/upx-3.96-amd64_linux/upx /usr/bin/ && rm -rf /tmp/upx-3.96-amd64_linux; \
     fi
 # -- for make rules
 
 # -- for dapper
 ENV DAPPER_RUN_ARGS --privileged --network host
 ENV GO111MODULE=on
-ENV CROSS=false
-ENV DAPPER_ENV CROSS CLUSTER_TYPE DOCKER_USERNAME DOCKER_PASSWORD WITHOUT_MANIFEST ONLY_MANIFEST IGNORE_MISSING DRONE_TAG REPO TAG OS ARCH IMAGE_NAME DIRTY_CHECK
+ENV CATTLE_DEV_MODE=true
```
Comment (JacieChao) [inline Dockerfile.dapper:33]: I think AUTOK3S_DEV_MODE will be better

**Review by JacieChao — COMMENTED** [review]
Others LGTM

File: hack/make-rules/autok3s.sh (hunk 1 above)
Comment (orangedeng) [reply hack/make-rules/autok3s.sh:159]: The `race` tags for go test requires CGO enable.

File: Dockerfile.dapper (hunk 2 above)
Comment (orangedeng) [reply Dockerfile.dapper]: Updated

**Review by JacieChao — APPROVED** [review]
LGTM

**Pull request merged**